superbench/runner/playbooks/deploy.yaml

```diff
@@ -100,6 +100,7 @@
           {{ '--gpus=all' if nvidia_gpu_exist else '' }} \
           {{ '--security-opt seccomp=unconfined --group-add video' if amd_gpu_exist else '' }} \
           -w /root -v {{ workspace }}:/root -v /mnt:/mnt \
+          -v /var/run/docker.sock:/var/run/docker.sock -v {{ workspace }}/docker:/var/lib/docker \
```

Comment:
only `docker.sock` is needed, check https://docs.nvidia.com/datacenter/cloud-native/playground/dind.html

Reply:
Removed.